client-java/controller-api/src/main/java/org/evomaster/client/java/controller/api/dto/database/operations/InsertionResultsDto.java

```diff
@@ -0,0 +1,19 @@
+package org.evomaster.client.java.controller.api.dto.database.operations;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class InsertionResultsDto {
+
+    /**
+     * primary key generated with the insertions
+     */
+    public Map<Long, Long> idMapping = new HashMap<>();
+
+    /**
+     * whether the insertion at the index executed successfully
+     */
+    public List<Boolean> executionResults = new ArrayList<>();
```

Comment:
specify that the list is ordered by index... of what exactly? the actions in the tests? or a specific series of SQL commands?